setup.py

```diff
@@ -201,8 +232,16 @@ def run(self):
     cmdclass=CMDCLASS,
 )
 
+# Qt bindings requirements
+qt_requirements = {
+    'pyqt5' : ['pyqt5>=5.15,<5.16', 'pyqtwebengine>=5.15,<5.16'],
+    'pyqt6' : ['pyqt6>=6,<7', 'pyqt6-webengine>=6,<7'],
```

Comment:
```suggestion
    'pyqt6' : ['pyqt6>=6.5,<7', 'pyqt6-webengine>=6.5,<7'],
```

This is our minimal supported version of pyqt6.